docs/advanced/working_with_git.rst

```diff
@@ -0,0 +1,207 @@
+================
+Working with git
+================
+
++-------------------------------+--------------------------------------------------------------------------------------+
+| Git Commands                  |                What it does                                                          |
++===============================+======================================================================================+
+| git **clone**  <repo>         | Used to clone the repo ``git clone <repo> | <name>``                                 |
++-------------------------------+--------------------------------------------------------------------------------------+
+| git **commit**                |  **Commit** an applied change on the given branch                                    |
++-------------------------------+--------------------------------------------------------------------------------------+
+| git **remote**                | `track <https://help.github.com/articles/configuring-a-remote-for-a-fork/>`_         |
+|                               | remote branch                                                                        |
++-------------------------------+--------------------------------------------------------------------------------------+
+| ``git revert <commit-SHA-1>`` |`Revert <http://git-scm.com/docs/git-revert>`_ changes in a commit                    |
++-------------------------------+--------------------------------------------------------------------------------------+
+|``git fetch <remote> <branch>``| The git `fetch <https://www.atlassian.com/git/tutorials/syncing/git-fetch/>`_ command|
+|                               | imports commits or `tags <http://git-scm.com/book/en/v2/Git-Basics-Tagging>`_        |
+|                               | from a remote repository into your local repo.                                       |
++-------------------------------+--------------------------------------------------------------------------------------+
+| **git pull**                  | **Updates** your repo. Shorthand for ``git fetch`` followed by                       |
+|                               | ``git merge FETCH_HEAD``. Recommended                                                |
+|                               | to use it with `- -rebase <http://bit.ly/1yAi5bf>`_.                                 |
```

Comment:
There is an extra space. It should be `--rebase

Also, I would remove the bit.ly link and use the source link. You may need to  create a new line after "with" to make readability easier and line length shorter.